provisioning_client/deps/CMakeLists.txt

```diff
@@ -14,51 +14,78 @@ else()
 
     if(${hsm_type_x509})
         set(msr_riot_c_files
-            ./RIoT/Emulator/DICE/DiceSha256.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotAes128.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotAesTables.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotCrypt.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotDerEnc.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotEcc.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotHmac.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotKdf.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotSha256.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotX509Bldr.c
-            ./RIoT/Emulator/RIoT/RIoTCrypt/RiotBase64.c
+            ./RIoT/Simulation/DICE/DiceSha256.c
```

Comment:
No change necessary here but a "consider for the future" would be to swap over to the Reference tree and away from the Simulation tree.  The sources are the same but the organization is a little different and the sets of sources serve slightly different purposes.

Reply:
Noted